stdlib/3/multiprocessing/context.pyi

```diff
@@ -0,0 +1,5 @@
+# Stubs for multiprocessing.context
+
+# NOTE: These are incomplete!
+
+class BaseContext(object): ...
```

Comment:
All good.  Reverted ctx to any and added a TODO.  I'll make a separate PR and add BaseContext in the next couple days or so.